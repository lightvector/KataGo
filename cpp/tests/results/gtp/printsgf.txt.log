: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/printsgf.txt.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
rules = tromp-taylor
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/printsgf.txt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: searchRandSeed = forTesting
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: printsgf
: = (;FF[4]GM[1]SZ[19]PB[]PW[]HA[0]KM[7.5]RU[TrompTaylor])
: Controller: boardsize 9
: GPU -1 finishing, processed 0 rows 0 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 9 * 9 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 9 nnYLen 9
: Initializing board with boardXSize 9 boardYSize 9
: = 
: Controller: kata-set-rules japanese
: Changed rules to Japanese
: = 
: Controller: play b c3
: = 
: Controller: play w c6
: = 
: Controller: play b c9
: = 
: Controller: printsgf -
: = (;FF[4]GM[1]SZ[9]PB[]PW[]HA[0]KM[7.5]RU[Japanese];B[cg];W[cd];B[ca])
: Controller: clear_board
: = 
: Controller: rectangular_boardsize 9:13
: GPU -1 finishing, processed 0 rows 0 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 9 * 13 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 9 nnYLen 13
: Initializing board with boardXSize 9 boardYSize 13
: = 
: Controller: kata-set-rules chinese
: Changed rules to Chinese
: = 
: Controller: place_free_handicap 3
: = C10 E10 D4
: Controller: genmove w
: MoveNum: 0 HASH: EFAE14B9FBCD6883AF04D1B9FDE22369
   A B C D E F G H J
13 . . . . . . . . .
12 . . . . . . . . .
11 . . . . . . . . .
10 . . X . X . . . .
 9 . . . . . . . . .
 8 . . . . . . . . .
 7 . . . . . . . . .
 6 . . . . . . . . .
 5 . . . . . . . . .
 4 . . . X . @ . . .
 3 . . . . . . . . .
 2 . . . . . . . . .
 1 . . . . . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 100
NN batches: 100
NN avg batch size: 1
PlayoutDoublingAdvantage: 2.625
PV: F4 G10 C9 D9 B10 C11 C8 C6 B11 B12
Tree:
: T -83.20c W -77.86c S  -4.03c (-10.3 L -10.4) N     100  --  F4 G10 C9 D9 B10 C11 C8
---White(^)---
F4  : T -82.96c W -77.25c S  -4.17c (-10.4 L -10.5) LCB  -89.44c P 55.50% WF  59.1 PSV      59 N      59  --  F4 G10 C9 D9 B10 C11 C8 C6
F10 : T -84.28c W -79.36c S  -3.62c ( -9.9 L -10.3) LCB  -91.97c P 14.68% WF  13.9 PSV      12 N      14  --  F10 F11 G11 E11 F4 F9
E4  : T -85.56c W -79.25c S  -4.18c (-10.4 L -10.6) LCB  -94.28c P 14.41% WF  12.9 PSV       9 N      13  --  E4 E3 F3 D3 F4 D6
F3  : T -81.41c W -78.12c S  -3.65c (-10.0 L -10.3) LCB  -91.77c P  4.73% WF  13.1 PSV       8 N      13  --  F3 F4 G4 F5 E3 G5

: = F4
: Controller: printsgf
: = (;FF[4]GM[1]SZ[9:13]PB[]PW[]HA[3]KM[7.5]RU[Chinese]AB[cd][ed][dj];W[fj])
: Controller: kata-set-rule tax seki
: Changed rules to koSIMPLEscoreAREAtaxSEKIsui0whbNfpok1
: = 
: Controller: printsgf
: = (;FF[4]GM[1]SZ[9:13]PB[]PW[]HA[3]KM[7.5]RU[koSIMPLEscoreAREAtaxSEKIsui0whbNfpok1]AB[cd][ed][dj];W[fj])
: GPU -1 finishing, processed 103 rows 103 batches
: All cleaned up, quitting
