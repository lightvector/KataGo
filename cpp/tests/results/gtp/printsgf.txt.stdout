= (;FF[4]GM[1]SZ[19]PB[]PW[]HA[0]KM[7.5]RU[TrompTaylor])

= 

= 

= 

= 

= 

= (;FF[4]GM[1]SZ[9]PB[]PW[]HA[0]KM[7.5]RU[Japanese];B[cg];W[cd];B[ca])

= 

= 

= 

= C10 E10 D4

= F4

= (;FF[4]GM[1]SZ[9:13]PB[]PW[]HA[3]KM[7.5]RU[Chinese]AB[cd][ed][dj];W[fj])

= 

= (;FF[4]GM[1]SZ[9:13]PB[]PW[]HA[3]KM[7.5]RU[koSIMPLEscoreAREAtaxSEKIsui0whbN]AB[cd][ed][dj];W[fj])

